site/docs/platforms.md

```diff
@@ -156,6 +156,13 @@ $ bazel build --platforms=//:myplatform //...`
 $ bazel build --platforms=//:myplatform //:all`
 ```
 
+Note that incompatible tests in a [`test_suite`](be/general.html#test_suite)
```

Comment:
Minor pedantic grammatical feedback: apply or not at your discretion:

I've heard guidance to try to avoid "note that" since what it communicates is generally clear enough from the rest of the sentence. i.e. just having the sentence implies it's worth noting.

I admit sometimes I still find it valuable to add "note that", so consider as appropriate.

Reply:
That's totally fair. I very much appreciate your insight here!

I applied your suggestion.